src/test/java/salesforcetest/steps/OpportunityStep.java

```diff
@@ -103,7 +103,7 @@ public void deleteOpportunities() {
     /**
      * Navigates to an opportunity.
      */
-    @And("I navigate to Opportunities Page")
+    @When("I navigate to Opportunities Page")
     public void navigatesToOpportunitiesPage() {
         PageTransporter pageTransporter = new PageTransporter();
         pageTransporter.navigateToPage(OPPORTUNITY_PAGE);
```

Comment:
here you need to send the page throug the scenario

Reply:
in this step I only change the decorator **@When**